.github/workflows/sanity-productV2.yml

```
@@ -0,0 +1,60 @@
+name: UI tests
+on:
+  pull_request:
+    paths:
+      - .github/workflows/sanity-productV2.yml
+      - admin-dev/themes/new-theme/js/pages/product/**
+      - src/PrestaShopBundle/Form/Admin/Sell/Product/**
+      - src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/**
+      - src/Core/Domain/Product/**
+      - src/Adapter/Product/**
+      - src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/**
```

```suggestion
      - src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/**
      - tests/UI/campaigns/productV2/**
```

Shouldn't it watch the test campaigns as well?

---

we could yes